Lib/test/test_plistlib.py

```diff
@@ -971,12 +971,12 @@ def test_cycles(self):
         self.assertIs(b['x'], b)
 
     def test_deep_nesting(self):
-        for N in [300, 100000]:
+        for N in [50, 300, 100000]:
```

Comment:
If we are going to be changing this line anyway ...
```suggestion
        for N in [50, 300, 100_000]:
```